loader/app/load/main.py

```diff
@@ -1,51 +1,91 @@
 import logging
 from datetime import datetime
+from json import JSONDecodeError
 
-import pandas
-from pandas import DataFrame
-
-from app.load.util import get_geography_id, get_type_id, post_action, policy_repr
-from app.mapping import CCLWActionType
+from app.load.api_client import get_type_id, get_geography_id, post_action
+from app.load.language import get_language_id_for_doc
+from app.model import PolicyLookup
 
 logger = logging.getLogger(__file__)
 
 
-def load(policies: DataFrame):
-    for idx, policy in policies.iterrows():
-        country_code = policy["country_code"]
+def load(policies: PolicyLookup):
+    imported_count = 0
+    for key, policy_data in policies.items():
+
+        country_code = key.country_code
         geography_id = get_geography_id(country_code)
         if not geography_id:
-            logger.warning(f"No geography found in lookup for country code {country_code}")
+            logger.warning(
+                f"No geography found in lookup for country code {country_code}"
+            )
             continue
 
-        policy_type = policy["policy_type"]
-        mapped_policy_type = CCLWActionType[policy_type].value
-        action_type_id = get_type_id(mapped_policy_type)
+        policy_type = key.policy_type
+        action_type_id = get_type_id(policy_type)
         if not action_type_id:
-            logger.warning(f"No action type found in lookup for policy type {policy_type}")
+            logger.warning(
+                f"No action type found in lookup for policy type {policy_type}"
+            )
             continue
 
-        if pandas.isnull(policy['policy_date']):
-            logger.warning("Date is null for policy", policy)
-
-        policy_date: datetime = policy['policy_date']
+        policy_date: datetime = key.policy_date
+        if policy_date is None:
+            logger.warning("Date is null for policy", key)
 
         action_payload = {
-            "name": policy["policy_name"],
-            "description": policy["description"],
+            "name": key.policy_name,
+            "description": policy_data.policy_description,
             "year": policy_date.year,
             "month": policy_date.month,
             "day": policy_date.day,
             "geography_id": geography_id,
             "type_id": action_type_id,
             "source_id": 1,  # CCLW is source_id 1
-            "documents": [],
+            "documents": [
+                {
+                    "name": doc.doc_name,
+                    "language_id": get_language_id_for_doc(doc),
+                    "source_url": doc.doc_url,
+                    "s3_url": None,  # TODO is this supposed to be doc.doc_url?
```

Comment:
@opyate the original idea for `s3_url` was that this stores the url of the document on s3 after it has been uploaded to an s3 bucket. It provides a link between the db entry for the action-document and the associated file on s3. `source_url` stores the original url of where the document was obtained from (e.g. CCLW or another source url)

Reply:
> f so, we'll need to make sure that we tackle this.

Sure, I'll start on #230 once this PR is merged.